tests/scripts/github-action-helper.sh

```diff
@@ -622,6 +622,23 @@ function test_csi_nfs_workload {
   kubectl exec -t pod/csinfs-demo-pod -- ls -alh /var/lib/www/html/
 }
 
+function install_minikube_with_none_driver() {
+  sudo apt update
+  sudo apt install -y conntrack socat
+  curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+  sudo dpkg -i minikube_latest_amd64.deb
+  rm -f minikube_latest_amd64.deb
+  curl -LO https://github.com/Mirantis/cri-dockerd/releases/download/v0.3.0/cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+  sudo dpkg -i cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+  rm -f cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+  wget https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.26.0/crictl-v1.26.0-linux-amd64.tar.gz
```

Comment:
How about we factor out the v1.26.0 as a variable, and move it to the top of this function along with the minikube version, so we can consider updating those together in the future?

Reply:
done